packages/api-gen/src/jsonOverrideUtils.ts

```diff
@@ -71,7 +71,13 @@ export async function loadJsonOverrides({
       const jsonOverridesFile = await readFileSync(pathToResolve, {
         encoding: "utf-8",
       });
-      const content = json5.parse(jsonOverridesFile);
+      const content = json5.parse(jsonOverridesFile, (key, value) => {
```

Comment:
can we move this to outside helper file and add tests?
So then we have documentation in tests what kind of results we're expecting in case of any errors

Reply:
yeah, good idea - I'm on it rn! 
should I also apply the same callback for any `json5.parse` invokation? Now I can see I missed some of the places, but to be honest, the local `*overrides.json` is the only place where this special `_DELETE_` mapping can be a case, or not? 🤔